server/src/main/proto/spine/system/server/command_lifecycle.proto

```diff
@@ -46,7 +46,9 @@ message CommandLifecycle {
     option (entity).kind = AGGREGATE;
 
     // The ID of the command.
-    spine.core.CommandId id = 1;
+    //
+    // The ID is not required since it is set only after the command was received.
```

Comment:
This is an issue, I think. Let's discuss.

Reply:
I have created the issue #877 since it isn't easy to make it required now.